packages/app/src/cli/commands/app/scaffold/extension.ts

```diff
@@ -64,27 +67,43 @@ export default class AppScaffoldExtension extends Command {
 
   public async run(): Promise<void> {
     const {flags} = await this.parse(AppScaffoldExtension)
+
+    await metadata.addPublic(() => ({
+      cmd_scaffold_required_auth: false,
+      cmd_scaffold_template_custom: flags['clone-url'] !== undefined,
+      cmd_scaffold_type_owner: '@shopify/app',
+    }))
+
     const directory = flags.path ? path.resolve(flags.path) : process.cwd()
     const app: AppInterface = await loadApp(directory)
 
     flags.type = mapExternalExtensionTypeToExtensionType(flags.type as ExternalExtensionTypes)
 
     await this.validateExtensionType(flags.type)
     this.validateExtensionTypeLimit(app, flags.type)
-    const extensionFlavor = flags.template
-    this.validateExtensionFlavor(flags.type, extensionFlavor)
+    this.validateExtensionFlavor(flags.type, flags.template)
 
     const promptAnswers = await scaffoldExtensionPrompt({
       extensionType: flags.type,
       extensionTypesAlreadyAtQuota: this.limitedExtensionsAlreadyScaffolded(app),
       name: flags.name,
-      extensionFlavor,
+      extensionFlavor: flags.template,
       directory: path.join(directory, 'extensions'),
     })
 
+    const {extensionType, extensionFlavor} = promptAnswers
+    await metadata.addPublic(() => ({
+      cmd_scaffold_template_flavor: extensionFlavor,
+      cmd_scaffold_type: extensionType,
+      cmd_scaffold_type_family: extensionTypeCategory(extensionType),
```

Comment:
family vs category, should we use the same naming?

Reply:
good point!